pyanaconda/ui/tui/hubs/__init__.py

```diff
@@ -153,6 +153,9 @@ def prompt(self, args=None):
         prompt = super().prompt(args)
 
         if self._spoke_count == 1:
-            prompt.add_option("1", _("to enter the %(spoke_title)s spoke") % {"spoke_title": list(self._spokes.values())[0].title})
+            prompt.add_option(
+                "1",
+                _("to enter the %(spoke_title)s spoke") % {"spoke_title": next(iter(self._spokes.values())).title}
```

Comment:
Honestly, this doesn't look like improvement. It's harder to read and understand.

Reply:
So keep this rule disabled ?